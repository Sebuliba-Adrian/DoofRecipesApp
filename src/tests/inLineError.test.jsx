import React from 'react';
import renderer from 'react-test-renderer';
import InLineError from '../components/authentication/InLineError';

it('renders correctly', () => {
  const inLineError = renderer.create(<InLineError text="err.." />).toJSON();
  expect(inLineError).toMatchSnapshot();
});
