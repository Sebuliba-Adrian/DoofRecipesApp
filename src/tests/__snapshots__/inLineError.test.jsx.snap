// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  style={
    Object {
      "color": "#ae5856",
    }
  }
>
  err..
</span>
`;
